### ISSUE TITLE
happiness

### ISSUE THREAD

**Issue by ilteroi** [LoneGazebo/Community-Patch-DLL#3898]
i know i'm hard to please ... but 100 happiness seems excessive.

![image](https://user-images.githubusercontent.com/1358966/35299160-b7630378-0084-11e8-9298-46cfdf436486.png)

![image](https://user-images.githubusercontent.com/1358966/35299165-c0237006-0084-11e8-9b33-4a6074cde90c.png)

and this is how they do it:

![image](https://user-images.githubusercontent.com/1358966/35299708-6553875e-0086-11e8-89e8-2728bbe18a58.png)

basically no unhappiness, except for one puppet suffering from boredom.

**Comment by Infixo** [comment]
In my test game today I had Japan with 17 cities. Reached Atomic era. 116 Happiness, accumulated 120000 (!) Gold. Basically entire infrastructure built in all cities except 5 puppets. Cities were running Food Process mostly.
